generics/services/samiksha-service.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * name : samiksha-service.js
+ * author : Rakesh Kumar
+ * Date : 4-June-2020
+ * Description : All samiksha service related api call.
+ */
+
+//dependencies
+
+let samikshaServiceBaseURL =
+    process.env.SAMIKSHA_SERIVCE_HOST +
+    process.env.SAMIKSHA_SERIVCE_BASE_URL +
+    process.env.URL_PREFIX;
+
+const request = require('request');
+const fs = require('fs');
+
+
+/**
+ * Entiies bulk upload 
+ * @name uploadFileToCloud
```

Comment:
@rakeshSgr - Name is wrong and not matching with function name

Reply:
resolved